package.json

```diff
@@ -18,32 +18,32 @@
   "license": "MIT",
   "homepage": "https://github.com/carforyou/carforyou-eslint-config-pkg#readme",
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.0.1",
-    "@typescript-eslint/eslint-plugin-tslint": "^4.0.0",
-    "@typescript-eslint/parser": "^4.0.0",
-    "eslint": "^7.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.1.1",
+    "@typescript-eslint/eslint-plugin-tslint": "^4.1.1",
+    "@typescript-eslint/parser": "^4.1.1",
+    "eslint": "^7.9.0",
     "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-cypress": "^2.10.3",
+    "eslint-plugin-cypress": "^2.11.1",
     "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jest": "^23.11.0",
-    "eslint-plugin-jsdoc": "^30.0.0",
-    "eslint-plugin-json": "^2.1.1",
-    "eslint-plugin-prefer-arrow": "^1.2.1",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-react": "^7.20.0",
-    "eslint-plugin-react-hooks": "^4.0.2",
+    "eslint-plugin-jest": "^24.0.1",
+    "eslint-plugin-jsdoc": "^30.5.1",
+    "eslint-plugin-json": "^2.1.2",
+    "eslint-plugin-prefer-arrow": "^1.2.2",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.20.6",
+    "eslint-plugin-react-hooks": "^4.1.2",
     "eslint-plugin-testing-library": "^3.1.2",
```

Comment:
need to keep this one down because of a peer dependency conflict introduced in later versions